bin/tictactoe

```diff
@@ -1,3 +1,56 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+def welcome
+  puts "Welcome to Tic Tac Toe!"
+  puts "Please choose a play mode"
+  puts "Enter 2 for two players"
+  puts "Enter 1 to play against the computer"
+  puts "Enter 0 to sit back and let the robots do the heavy lifting."
+  player_types
+end
+
+def player_types
+  input=gets.strip
+  if input=="2"
+   game= Game.new
+    game.play
+  elsif input=="1"
+   which_player
+ elsif input=="0"
+  game=Game.new(Player::Computer.new("X"), Player::Computer.new("O"), Board.new)
+  game.play
+else
+  puts "Please enter a '2', '1', or '0'."
+  player_types
+end
+play_again
+end
+
+
+
+   def which_player
```

Comment:
Try to be consistent with your indentation; it will help keep your code clean and readable.